app/assets/stylesheets/layouts/starrydark.scss

```diff
@@ -1,157 +1,287 @@
+$bg_color_header_bottom: #484357; // bottom header strip with navigation
+
 #header {
-  background-color: #484357;
+  background-color: $bg_color_header_bottom;
   background-image: url(image-path("layouts/starry.jpg"));
   background-position: right top;
 }
-#logo, #nav-bottom { background-color: transparent !important; }
-#user-info, #nav-top { background-color: #05050F; background-color: rgba(5, 5, 15, 0.9); }
-
-$FONT_COLOR: #DADADA;
-$FONT_COLOR_DARK: #444444;
-body { background-color: #211E2F; color: $FONT_COLOR; }
-
-th, .content-header { background-color: #0D0B1E; }
-.sub { color: #F3F3F3; }
-.sub, .post-navheader, .flash.breadcrumbs { background-color: #131937; }
-.form-table-ender { background-color: #0D0B1E; color: $FONT_COLOR; }
-.selector-popup, .single-description, .subber { background-color: #484357; color: $FONT_COLOR; }
-th, .single-description, .content-header {
-  a { color: #B5AFCB; }
-  a:visited { color: #9C9AA4; }
+#logo { background-color: $bg_color_header_top !important; }
+#user-info, #nav-top { background-color: $bg_color_header_middle; background-color: rgba($bg_color_header_middle, 0.9); }
+
+/* - links */
+$font_color_link: #B5AFCB; // link
+$font_color_link_visited: #9C9AA4;
+
+a { color: $font_color_link; }
+a:visited { color: $font_color_link_visited; }
+
+/* - flashes */
+// these are the notifications that appear when you submit a form or get a message
+// TODO: See if we can differentate these
+$bg_color_flash_error: #484357; // error
+$bg_color_flash_success: #484357; // success
+$bg_color_flash_inbox: #484357; // new message
+$bg_color_flash_breadcrumbs: #131937; // breadcrumbs ("Continuities » Continuity » Section » Thread")
+$font_color_flash: #DADADA;
+$font_color_breadcrumbs_bold: #F3F3F3;
+
+.flash.breadcrumbs {
+  background-color: $bg_color_flash_breadcrumbs;
+  &, a, a:visited { color: $font_color_link_visited; }
+  b { color: $font_color_breadcrumbs_bold; } // TODO: Investigate why this is differentated
 }
+.flash.success { background-color: $bg_color_flash_success; color: $font_color_flash; }
+.flash.error { background-color: $bg_color_flash_error; color: $font_color_flash; }
+.flash.inbox { background-color: $bg_color_flash_inbox; color: $font_color_flash; }
 
-$EVEN_COLOR: #31323B;
-$ODD_COLOR: #25262E;
+/* - main page */
+$bg_color_page: #211E2F; // overall page
+$font_color_page: #DADADA;
 
-#post-menu-box {
-  border-top: 1px solid #808080;
-  border-top: 1px solid rgba(128,128,128,0.5);
+body { background-color: $bg_color_page; color: $font_color_page; }
 
-  a { color: inherit; }
-  div {
-    border-bottom: 1px solid #808080;
-    border-bottom: 1px solid rgba(128,128,128,0.5);
-  }
+$bg_color_spacer_alt: #131937;
+$font_color_search_box: #FEFEFE;
+$bg_color_search_box: #131937;
+
+.search-box { background-color: $bg_color_search_box; color: $font_color_search_box; }
+.spacer-alt { background-color: $bg_color_spacer_alt }
```

Comment:
This has a `color:` declaration in `starrydark.css` too, maybe re-add it here?

Reply:
I think I removed that b/c that should never contain text?

Reply:
That's fair. Can you put a semicolon after `bg_color_spacer_alt`, then?